experimental/SymmetricIntersections/src/homogeneous_polynomial_actions.jl

```diff
@@ -205,7 +205,7 @@ function symmetric_intersections(prep::ProjRep, d::Int, t::Int; j::Union{MapFrom
   elseif check
     V = domain(j)
     @req base_ring(V) === F "Incompatible map j"
-    @req dim(V) == binomial(dimension_representation(prep)+d-1, d) "Incompatible map j"
+    @req rank(V) == binomial(dimension_representation(prep)+d-1, d) "Incompatible map j"
```

Comment:
So what is `V` here? the docstring suggests it is a vector space, so maybe `vector_space_dim` should be used? Or it should stay at `dim`?